tests/experimental/datasources/test_pandas_datasource.py

```diff
@@ -1,14 +1,16 @@
 import os
+import pathlib
```

Comment:
I do prefer just importing `pathlib` and then using `pathlib.Path`. 👍 
There's actually a fair number of third party packages that use a `Path` object so this makes things explicit.